mlflow/server/handlers.py

```diff
@@ -262,6 +262,43 @@ def wrapper(*args, **kwargs):
 ]
 
 
+def _disable_mlflow_artifacts_endpoint(func):
```

Comment:
```suggestion
def _disable_unless_serve_artifacts(func):
# I'm not sure if this sounds fine...
```

This is not a big issue, but can we rename the function to indicate that the endpoint disablement is conditional?

Reply:
I like it. Refactored.